docker-hub/registry-access-management.md

```diff
@@ -0,0 +1,114 @@
+---
+description: Registry Access Management
+keywords: registry, access, managment
+title: Registry Access Management
+---
+
+Registry Access Management is a new feature available to organizations with a Docker Business subscription. This feature lets organization owners manage the registries that their users can access while using Docker Desktop.  When using this feature, organization owners can ensure that their users can only access their trusted registries, such as a secure private registry on Artifactory, thereby reducing the security risks that can occur when users interact with public registries.
+
+## Configure Registry Access Management permissions
+
+To configure Registry Access Management permissions, perform the following steps:
+
+1. Sign into your [Docker Hub](https://hub.docker.com) account as an organization owner.
+2. Select an organization, navigate to the **Settings** tab on the **Organizations** page and click **Org Permissions**.
+3. Enable Registry Access Management to set the permissions for your registry.
+
+> **Note**
+>
+> When enabled, the Docker Hub registry is set by default, however you can also restrict this registry for your developers.
+
+4. Click **Add** and enter your registry details in the applicable fields, and click **Create** to add the registry to your list.
+5. Verify that the registry appears in your list and click **Save & Apply**.  You can verify that your changes are saved in the Activity tab.
+
+  > **Note**
+  >
+  > Once you add a registry, it can take a maximum of 24 hours for the changes to be enforced on your developers’ machines. If you want to apply the changes sooner, you must force a Docker logout on your developers’ machine and have the developer re-authenticate for Docker Desktop.  Also, there is no limit on the number of registries you can add. See the Caveats sections to learn more about limitations  when using this feature.
```

Comment:
```suggestion
  > Once you add a registry, it can take up to 24 hours for the changes to be enforced on your developers’ machines. If you want to apply the changes sooner, you must force a Docker logout on your developers’ machine and have the developers re-authenticate for Docker Desktop.  Also, there is no limit on the number of registries you can add. See the Caveats sections to learn more about limitations  when using this feature.
```

Reply:
Could you add a link to the Caveats section?

Reply:
Updated